Amazon.QLDB.Driver.Tests/AsyncQldbDriverTests.cs

```diff
@@ -265,15 +263,12 @@ public async Task TestAsyncExecuteWithFuncLambdaAndRetryPolicyReturnsFuncOutput(
             catch (Exception e)
             {
                 Assert.IsTrue(expectThrow);
```

Comment:
Is this still needed? Isn't Count != 0 the same, and you can get rid of expectThrow?